gitpod.md

```diff
@@ -0,0 +1,47 @@
+# Hack on Real World Polylith in your Browser
+
+Using [Gitpod](https://www.gitpod.io) you can explore a [Polylith](https://polylith.gitbook.io/polylith/) implementation of [Real World](https://www.realworld.how/) from the Clojure REPL, without downloading or installing anything at all.
+
+![Alt text](.media/gitpod/Gitpod-Polylith-RealWorld.png)
+
+Don't click on this link just yet: 
+https://gitpod.io/#https://github.com/furkan3ayraktar/clojure-polylith-realworld-example-app
```

Comment:
Instead of saying, `Don't click`, should we just move this link to below the line that says `Happy coding! ❤️`

Reply:
I want the reader to know about this link when reading the instructions. And to find it easily, should they close the tab after having read the instructions and return to the page later when they feel they have the time to spend.